src/engraving/dom/score.cpp

```diff
@@ -4543,6 +4543,28 @@ ChordRest* Score::findCRinStaff(const Fraction& tick, staff_idx_t staffIdx) cons
     return 0;
 }
 
+ChordRest* Score::findChordRestEndingBeforeTickInTrack(const Fraction& tick, track_idx_t trackIdx) const
+{
+    Measure* measure = tick2measureMM(tick - Fraction::eps());
```

Comment:
Could you please explain why we need to subtract eps() here?

Reply:
Because when tick falls exactly in-between two measures (meaning that one measure ends at tick and the next one starts at tick) tick2Measure returns the next one (which is what we want in 99% of cases). I'm subtracting eps() to make sure that I get the _previous_ one in this case